app.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Slackin",
+  "description": "",
```

Comment:
I just added *A little server that enables public access to a Slack server*, taken from the readme's own introduction.